scripts/controlnet.py

```diff
@@ -388,8 +388,7 @@ def get_element(obj, strict=False):
                 return None
 
         attribute_value = get_element(getattr(p, attribute, None), strict)
-        default_value = get_element(default)
```

Comment:
The default value here now can be list. Calling `get_element` will get the item on `idx`, which is not what we want.

Generally I don't think here `get_element` call is justified on `default`.